src/Symfony/Component/Security/Core/User/UserInterface.php

```diff
@@ -78,4 +78,11 @@ public function getSalt();
      * the plain-text password is stored on this object.
      */
     public function eraseCredentials();
+
+    /**
+     * @return string
+     *
+     * @deprecated since Symfony 5.3, use getUserIdentifier() instead
```

Comment:
I fear this will trigger a deprecation notice that is impossible to fix for end users. Can you please confirm it does not before we merge?

Reply:
See the PR description. Is there a situation I didn't test you think will result in a deprecation?

Reply:
Thanks. I just checked out your PR, RAS.